transcom/move.mil
# Enable Permissions and Role-based permissions

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Export role-based permissions
- Address this ticket: https://www.pivotaltracker.com/story/show/158393206

## Testing

To verify the changes proposed in this pull request…

1. pull changes
1. `make cim`
1. Go to http://move.mil.localhost:8000/admin/people/permissions
1. Verify the permissions are the same ones that Feld specify in the ticket.
